package.json

```diff
@@ -14,7 +14,7 @@
     "build-storybook": "build-storybook -c .storybook -o .storybook-out",
     "storybook": "start-storybook",
     "test-unit": "jest -t",
-    "test-security": "python script/audit_deps.py",
+    "test-security": "python3 script/audit_deps.py .",
```

Comment:
I'm confused about this because it's not the same as https://github.com/brave/brave-browser/blob/master/package.json#L26
cc @mbacchi @mihaiplesa are we running this in CI or the brave-browser version of it? It seems like this should be an alias for the brave-browser test-security similar to https://github.com/brave/brave-core/blob/master/package.json#L7

Reply:
not running that anywhere, we did in the past

Reply:
It gets run in Travis: https://travis-ci.org/github/brave/brave-core/jobs/697019011